pinot-query-runtime/src/main/java/org/apache/pinot/query/mailbox/GrpcSendingMailbox.java

```diff
@@ -64,25 +69,31 @@ public void send(TransferableBlock block)
       return;
     }
     if (LOGGER.isDebugEnabled()) {
-      LOGGER.debug("==[GRPC SEND]== sending data to: " + _id);
+      LOGGER.debug("==[GRPC SEND]== sending message " + block + " to: " + _id);
     }
     if (_contentObserver == null) {
       _contentObserver = getContentObserver();
     }
     _contentObserver.onNext(toMailboxContent(block));
+    if (LOGGER.isDebugEnabled()) {
+      LOGGER.debug("==[GRPC SEND]== message " + block + " sent to: " + _id);
+    }
```

Comment:
I've found this logs very useful to debug a concurrency issue in this PR. All these logs are either not in the hot path or warded with `isDebugEnabled` in order to do not allocate if disabled.